src/util-device.h

```diff
@@ -50,7 +50,13 @@ typedef struct LiveDevice_ {
     uint32_t offload_orig;  /**< original offload settings to restore @exit */
 } LiveDevice;
 
+typedef struct PreLiveDevice_ {
```

Comment:
can we not have 'Pre' prefix? If you have to add something, make it a suffix please

Reply:
Renamed to LiveDeviceName as it is that currently